database/migrations/2019_05_09_164744_create_blog_table.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateBlogTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up(): void
+    {
+        Schema::create('categories', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name');
+            $table->timestamps();
+        });
+
+        Schema::create('articles', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('title');
+            $table->longText('content');
+            $table->bigInteger('author_id')->unsigned()->nullable();
+            $table->foreign('author_id')->references('id')->on('users')->onDelete('set null');
+            $table->unsignedInteger('category_id')->nullable();
+            $table->foreign('category_id')->references('id')->on('categories')->onDelete('set null');
+            $table->timestamps();
+        });
+
+
+        Schema::create('comments', function (Blueprint $table) {
+            $table->increments('id');
+            $table->longText('content');
+            $table->unsignedInteger('article_id')->nullable();
+            $table->foreign('article_id')->references('id')->on('articles')->onDelete('cascade');
+            $table->timestamps();
+        });
+
+        Schema::create('likes', function (Blueprint $table) {
```

Comment:
missing `$table->increments('id');`

Reply:
Why this id is necessary, we can't reach the likes of an user without ID ? Because in symfony (for example) we don't need an id

Reply:
it's always easier to have id on table